README.md

```diff
@@ -1,5 +1,8 @@
-# elementary GitHub Actions
+# Elementary GitHub Actions
+## What is it?
+Elementary GitHub Actions is an open repository to aid you in simple actions - things you may need in your everyday lives of code.
```

Comment:
Same as above with `elementary`. This sentence is also really just fluff; everything on GitHub is an open repository, and the rest doesn't really mean anything.